deps.edn

```diff
@@ -5,9 +5,11 @@
         generateme/fastmath {:mvn/version "2.4.0"}
         aerial.hanami/aerial.hanami {:mvn/version "0.20.0"}
         cljplot/cljplot {:mvn/version "0.0.3"}
-        ;; scicloj/scicloj.ml {:mvn/version "0.3"}
         scicloj/metamorph.ml {:mvn/version "0.7.8"}
         scicloj/scicloj.ml.smile {:mvn/version "7.2"}
+        scicloj/scicloj.ml.tribuo {:git/url "https://github.com/scicloj/scicloj.ml.tribuo"
```

Comment:
We need to use "Git" dependencies, as Artifactory/mvn dependencies do not work with tribuo-all:pom.
Known issue of Clojure: https://ask.clojure.org/index.php/10892/add-support-for-bom-bill-of-materials-dependencies?show=10925#a10925